pry.gemspec

```diff
@@ -29,4 +29,10 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'coderay',       '~> 1.1.0'
   s.add_dependency 'method_source', '~> 0.9.0'
+
+  if s.respond_to?(:metadata)
```

Comment:
Why is this needed?

Reply:
@kyrylo don't we generate our gemspec using 'rake gemspec' ?

Reply:
Seems so. I'm confused because we do have the information this PR specifies here 
<img width="220" alt="1" src="https://user-images.githubusercontent.com/1079123/48216368-81c17c80-e3bf-11e8-8c0e-84012c68385d.png">

So why do we need this PR?

https://rubygems.org/gems/pry

Reply:
I mainly added this to add the changelog link, which is missing. I can't find the gemspec generating part, but I'm happy to modify that instead if you point in the right direction.

Reply:
Why is the `respond_to?` call needed?

Reply:
For old versions of rubygems. Not sure what's the threshold, but if you serialise gemspec from a ruby object, the same check will be added.

Reply:
Is the old version of rubygems really a concern here? I thought the gemspec is being executed when you `build` a gem.

If you want to link to CHANGELOG, then let's just leave that one link. We already have the rest covered.